### ISSUE TITLE
Remove `--no-verify` from CI once github action is updated

### ISSUE THREAD

**Issue by digorithm** [FuelLabs/fuels-rs#81]
Once https://github.com/katyo/publish-crates/pull/394 is merged, we should be able to remove the `--no-verify` flag from our CI:

```
- name: Publish crate
      uses: katyo/publish-crates@v1
      with:
          publish-delay: 30000
          args: --no-verify
          registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

**Comment by Voxelot** [comment]
FYI @digorithm the upstream PR is now merged

**Comment by adlerjohn** [comment]
Re-opening due to #118

**Comment by adlerjohn** [comment]
Re-closed in #134
